css-three-column-layout/styles.css

```diff
@@ -0,0 +1,46 @@
+.container {
+  background-color: rgb(26 150 218);
+  color: white;
+  text-align: center;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  border-width: 40px;
+  padding: 40px 50px;
+  max-width: 1000px;
+  margin: auto;
+}
+
+.columns {
+  display: flex;
+}
+
+.column {
+  background-color: rgb(26 150 218);
+  color: white;
+  width: calc(100% / 3);
+  float: left;
+  padding: 0 15px;
+  margin: 0 15px;
```

Comment:
The only valid property in a `column-` class is `flex-basis`. Anything else should be on a separate class that you can add to the element in addition to the `column-` class.